src/main/java/com/epam/jdbc/service/serviceimpl/AcademicCourseServiceImpl.java

```diff
@@ -37,4 +37,22 @@ public AcademicCourse findByName(String academicCourseName) {
         }
         return academicCourse;
     }
+
+    @Override
+    public int findAcademicCourseIdByLinkedClassId(int academicClassId) {
+        logger.info("Find ID of the given academic course.");
+        int id = -1;
+        String query = "SELECT academic_course_id FROM public.academic_class_academic_course_mapping WHERE academic_class_id=?;";
```

Comment:
Please write like this -> 
SELECT academ......
FROM public...
WHERE aca='..'.

Reply:
resolved